contracts/standard/arbitration/CentralizedCourt.sol

```diff
@@ -300,7 +300,7 @@ contract CentralizedCourt is Arbitrator {
     function timeoutUpdatePriceRequest(uint _requestID) public {
         ArbitrationPriceRequest storage request = requests[_requestID];
         require(now >= request.deadline, "Can only timeout arbitration update price request after deadline.");
-        require(request.status == DisputeStatus.Waiting, "Can only timeout disputes not resolved.");
+        require(request.status == DisputeStatus.Waiting, "Can only timeout arbitration update price request not resolved.");
```

Comment:
"DisputeStatus.Waiting" still refers to disputes. We need to either change the enum name or make two different ones.

Reply:
Already corrected in commit this https://github.com/kleros/kleros-interaction/pull/90/commits/67902e77a7525a09ec0b48f02d1043e863bbdd69